src/assign-graders/utils.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-ignore
```

Comment:
This isn't ideal as the idea behind using typescript is a consistent and well typed project, I've fixed the errors in this file in my commit.

Reply:
I thought the library simply didn't provide types. Thanks!

Reply:
When the NPM page for the package has the logo of `DT` beside the package name, that means that there is a package that provides TS typing with the form of `@types/[package-name]`:
![image](https://user-images.githubusercontent.com/36013798/183066100-b74fc20e-b011-4ad2-a59f-42eba52bf8e6.png)